packages/protocol/lib/test-utils.ts

```diff
@@ -503,9 +532,11 @@ enum VoteValue {
   Yes,
 }
 
-export async function assumeOwnership(contractsToOwn: string[], to: string, proposalId: number = 1, dequeuedIndex: number = 0) {
+export async function assumeOwnership(contractsToOwn: string[], to: string, dequeuedIndex: number = 0, path='') {
```

Comment:
checking the one default to  zero default is correct?

Reply:
a subtle change that could have big impact of not intended fully

Reply:
yeap, it's correct!